core/src/main/res/values/strings.xml

```diff
@@ -321,4 +321,5 @@
   <string name="open_article">Open Article</string>
   <string name="delete_note_dialog_message">Note: Notes are not deleted from your storage</string>
   <string name="delete_selected_notes">Delete Selected Notes?</string>
+  <string name="missing_split_version_for_objectbox">Un-Supported architecture, your device isn\'t compatible with this app</string>
```

Comment:
I realise you've merged this code, nonetheless here's a suggested change to the wording as the device isn't compatible with this version of the app. 

`Un-Supported architecture, your device isn\'t compatible with this copy of the app`

For example, they may have copied the app from an incompatible binary format. As the FAQ notes https://developer.android.com/guide/app-bundle/faq#if_a_developer_uses_aabs_to_deliver_optimized_apks_does_this_mean_that_its_possible_for_users_who_share_those_apks_to_run_into_issues "In these situations, an APK or set of APKs that are appropriate for the target device need to be installed."